Coding-style-guide.md

```diff
@@ -18,25 +18,33 @@ If you use [Sublime Text](http://www.sublimetext.com/), consider installing the
 
 
 ## Python
+
+#### Black
```

Comment:
Please move the section that talks about Black (basically, all the new stuff) just above the section that talks about Prettier, and give it the same level of heading. We should be consistent in how we describe the formatters.

Reply:
Done.